### PR TITLE
Update public-en.md

### PR DESCRIPTION
**Pull request by STdevK**
NodeReal recently provide node services to Klaytn

## Proposed changes
To update the list of Klaytn node service provider

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Comment by STdevK** [comment]
recheck

**Comment by markyim-klaytn** [comment]
@STdevK Please just copy&paste below message.
**I have read the CLA Document and I hereby sign the CLA**

**Comment by STdevK** [comment]
@markyim-klaytn 
Screenshot 2023-03-09 at 3 35 41 PM

**Comment by STdevK** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/dapp/json-rpc/public-en.md [hunk 1]
```diff
@@ -84,6 +84,28 @@ Since we cannot guarantee uptime and stability of the endpoints, do not use them
...
+- Immediate finality.
+- High TPS that meets real-world use cases.
+- Lower the cost of running Blockchain Applications.
+- Lower the barriers to entry for end-users.
+- Ease the technology adoption process for industry.
+- 50 QPS/method
+
+#### References
+
+[Docs] https://docs.nodereal.io/docs/getting-started
```
Comment (toniya-klaytn) [inline docs/dapp/json-rpc/public-en.md:103]:
```suggestion
[Docs](https://docs.nodereal.io/docs/getting-started)
```

File: docs/dapp/json-rpc/public-en.md [hunk 2]
```diff
@@ -84,6 +84,28 @@ Since we cannot guarantee uptime and stability of the endpoints, do not use them
...
+[Subscription] https://nodereal.io/api-marketplace/klaytn-rpc
```
Comment (toniya-klaytn) [inline docs/dapp/json-rpc/public-en.md:104]:
```suggestion
[Subscription](https://nodereal.io/api-marketplace/klaytn-rpc)
```

File: docs/dapp/json-rpc/public-en.md [hunk 3]
```diff
@@ -84,6 +84,28 @@ Since we cannot guarantee uptime and stability of the endpoints, do not use them
...
+[Website] https://nodereal.io
```
Comment (toniya-klaytn) [inline docs/dapp/json-rpc/public-en.md:105]:
```suggestion
[Website](https://nodereal.io)
```

File: docs/dapp/json-rpc/public-en.md [hunk 4]
```diff
@@ -84,6 +84,28 @@ Since we cannot guarantee uptime and stability of the endpoints, do not use them
...
 By using node services, you can focus on developing your application or product on Klaytn instead of spending your resources on node operation. Below is a list of node providers.
 
+### NodeReal
+
+NodeReal is more than just a blockchain provider - we're your all-in-one destination for everything related to Web3! Our goal is to make blockchain accessible, scalable and secure for everyone, no matter where they are. We have an enthusiastic team that is passionate about blockchain technology, and we're committed to helping the ecosystem grow and prosper over the long term. Our clients rely on us to provide dependable, effective and leading-edge solutions, and we take pride in delivering outcomes that go beyond their expectations. Come join us in promoting the widespread adoption of Web3 and transforming the future of technology!
+
+#### Features
+
+NodeReal supports Klaytn's key design goals of:
+- Immediate finality.
```
Comment (toniya-klaytn) [inline docs/dapp/json-rpc/public-en.md:94]: The contents seems little irrelevant to the public node feature

File: docs/dapp/json-rpc/public-en.md [hunk 5]
```diff
@@ -84,6 +84,28 @@ Since we cannot guarantee uptime and stability of the endpoints, do not use them
 
 By using node services, you can focus on developing your application or product on Klaytn instead of spending your resources on node operation. Below is a list of node providers.
 
+### NodeReal
+
+NodeReal is more than just a blockchain provider - we're your all-in-one destination for everything related to Web3! Our goal is to make blockchain accessible, scalable and secure for everyone, no matter where they are. We have an enthusiastic team that is passionate about blockchain technology, and we're committed to helping the ecosystem grow and prosper over the long term. Our clients rely on us to provide dependable, effective and leading-edge solutions, and we take pride in delivering outcomes that go beyond their expectations. Come join us in promoting the widespread adoption of Web3 and transforming the future of technology!
```
Comment (toniya-klaytn) [inline docs/dapp/json-rpc/public-en.md:89]:
```suggestion
NodeReal is a blockchain infrastructure and services provider. NodeReal helps developers and investors to explore blockchains with the most reliable solutions.
```

File: docs/dapp/json-rpc/public-en.md (hunk 4 above)
Comment (ayo-klaytn) [reply docs/dapp/json-rpc/public-en.md:94]: You can reference the features in this [document](https://docs.nodereal.io/docs/pricing-plan#free-tier)

Comment (STdevK) [reply docs/dapp/json-rpc/public-en.md]: Thank you for the reference, I've changed the features

File: docs/dapp/json-rpc/public-en.md [hunk 6]
```diff
@@ -91,11 +91,9 @@ NodeReal is a blockchain infrastructure and services provider. NodeReal helps d
 #### Features
 
 NodeReal supports Klaytn's key design goals of:
```
Comment (ayo-klaytn) [inline docs/dapp/json-rpc/public-en.md:93]: can this sentence be removed?

File: docs/dapp/json-rpc/public-en.md [hunk 7]
```diff
@@ -91,11 +91,9 @@ NodeReal is a blockchain infrastructure and services provider. NodeReal helps d
...
-- Immediate finality.
-- High TPS that meets real-world use cases.
-- Lower the cost of running Blockchain Applications.
-- Lower the barriers to entry for end-users.
-- Ease the technology adoption process for industry.
+- Free Tier, 3 API keys, 350M CU monthly , 300 CUPS monthly, Archive data
```
Comment (ayo-klaytn) [inline docs/dapp/json-rpc/public-en.md:94]: can we have the abbreviations in full? something like: 3500 Compute Units (CU) and 300 Computing Units Per Second (CUPS)

File: docs/dapp/json-rpc/public-en.md (hunk 6 above)
Comment (STdevK) [reply docs/dapp/json-rpc/public-en.md]: Yes, consider it done,

File: docs/dapp/json-rpc/public-en.md (hunk 7 above)
Comment (STdevK) [reply docs/dapp/json-rpc/public-en.md]: Point taken and changes applied.

**Comment by toniya-klaytn** [comment]
@STdevK  Also can we have this section at the end of `Subscription-based JSON-RPC providers` section ? Instead of having it before KAS ?
cc: @ayo-klaytn

**Comment by STdevK** [comment]
@toniya-klaytn @ayo-klaytn I've shifted the paragraphs after KAS.

**Review by praveen-klaytn — APPROVED** [review]

**Review by ayo-klaytn — APPROVED** [review]

**Pull request merged**